outputs.tf

```diff
@@ -23,33 +23,3 @@ output "bigquery_tables" {
   value       = google_bigquery_table.main
   description = "Map of bigquery table resources being provisioned."
 }
-
-output "dataset_project" {
-  value       = google_bigquery_dataset.main.project
-  description = "Project where the dataset and table are created"
-}
-
-output "table_id" {
-  value       = google_bigquery_table.main.*.id
```

Comment:
Instead of dropping these outputs entirely, we should try to fix them. Similar for the others.

```
  value = {
    for table in google_bigquery_table. main:
    table.friendly_name => instance.table_id
  }
```

Reply:
So the reason I didn't bother fixing them was I believe they were accidentally re-added.

See the history of outputs.tf: https://github.com/terraform-google-modules/terraform-google-bigquery/commits/master/outputs.tf

In particular:
- Dec 2 which removed the extra outputs: https://github.com/terraform-google-modules/terraform-google-bigquery/commit/9dc3d31ac4a20d7c871723f326bacd937c8f3183#diff-c09d00f135e3672d079ff6e0556d957d
- Dec 15 which added them back accidentally: https://github.com/terraform-google-modules/terraform-google-bigquery/commit/c485f77ea7bd69d15cdb1eeef94d0d8943d51898#diff-c09d00f135e3672d079ff6e0556d957d

I would prefer the simpler approach of not adding all these ouputs since we return the list of tables the user can easily compute these values if they care to. But if you feel we should kep some of these I am happy to re-add.